pkg/segment/structs/segmeta.go

```diff
@@ -35,6 +35,7 @@ type VtableCounts struct {
 	OnDiskBytesCount uint64
 }
 
+// Ensure that when SegMeta is read, the corresponding SegFullMeta is also read
 type SegMeta struct {
```

Comment:
Why? I thought the point of splitting SegFullMeta into a different struct/file is so that when we only need info that's in the SegMeta, we can just read that info and not read a lot more info that's in SegFullMeta

Reply:
See the comment below.